baseapp/testutil/testpb/messages.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package testdata;
+package testpb;
```

Comment:
Another idea here is simply:
```suggestion
```

I think this would work too. Then, since there's no package, we can keep `messages.proto` in the testutil root folder

Reply:
Yes this works thanks!